src/Compiler/Service/ServiceDeclarationLists.fs

```diff
@@ -196,14 +196,13 @@ module DeclarationListHelpers =
         // Active pattern tags 
         | Item.ActivePatternCase apref -> 
             let v = apref.ActivePatternVal
-            // Format the type parameters to get e.g. ('a -> 'a) rather than ('?1234 -> '?1234)
             let vTauTy = v.TauType
             // REVIEW: use _cxs here
             let (prettyTyparInst, prettyTy), _cxs = PrettyTypes.PrettifyInstAndType denv.g (item.TyparInstantiation, vTauTy)
             let remarks = OutputFullName displayFullName pubpathOfValRef fullDisplayTextOfValRefAsLayout v
             let layout =
                 wordL (tagText (FSComp.SR.typeInfoActiveRecognizer())) ^^
-                wordL (tagActivePatternCase apref.Name |> mkNav v.DefinitionRange) ^^
+                wordL (tagActivePatternCase apref.LogicalName |> mkNav v.DefinitionRange) ^^
```

Comment:
This should fix quick info when hovering over `Thing with space` in 

```fsharp
let (|``Thing with space``|_) x = if x % 2 = 0 then Some (x/2) else Non

match 4 with
| ``Thing with space`` v -> "yes"
| _ -> "no"
```

Reply:
Actially, no, I'm wrong.  This is using `apref.LogicalName`, it should be using `apref.DisplayName`.  Please adjust

Reply:
Yes, now this is also fixed. [Ticket](https://github.com/dotnet/fsharp/issues/13610), [test](https://github.com/dotnet/fsharp/pull/13552/files#diff-1e6dc3c6f1f00f584bbc25dce0ef0259e996c14c91d1b58b58ace6e297325cabR518).